packages/fluentui/react-northstar/src/components/Datepicker/DatepickerCalendarHeader.tsx

```diff
@@ -192,6 +192,12 @@ DatepickerCalendarHeader.propTypes = {
   weekNumberFormatString: PropTypes.string,
   selectedDateFormatString: PropTypes.string,
   todayDateFormatString: PropTypes.string,
+  calendarCellFormatString: PropTypes.string,
```

Comment:
Please why do we need this props here, if they are not used anywhere? Or am I missing something?

Reply:
It's because we have flattened the localization props. 

This 'long-list' of prop types and default values actually really favors the 'composite' approach for localization strings.

Reply:
Addressed the comment.